packages/core/data-transfer/types/providers.d.ts

```diff
@@ -19,6 +19,8 @@ export interface ISourceProvider extends IProvider {
   streamLinks?(): NodeJS.ReadableStream | Promise<NodeJS.ReadableStream>;
   streamMedia?(): NodeJS.ReadableStream | Promise<NodeJS.ReadableStream>;
   streamConfiguration?(): NodeJS.ReadableStream | Promise<NodeJS.ReadableStream>;
+  getSchemas?(): any;
+  streamSchemas?(): NodeJS.ReadableStream;
```

Comment:
In some providers, we'll need this method to be async

```suggestion
  streamSchemas?(): NodeJS.ReadableStream | Promise<NodeJS.ReadableStream>;
```